buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/behaviorinterface.py

```diff
@@ -106,9 +156,9 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
         self.session_path = Path(self.source_data["folder_path"])
         self.session_id = self.session_path.stem
 
-        module_name = "Figure - 8 maze"
-        module_description = "Figure 8 - maze"
-        processing_module = get_module(nwbfile=nwbfile, name=module_name, description=module_description)
+        # module_name = "FigureEightMaze"
+        # module_description = "A figure-eight maze"
+        # processing_module = get_module(nwbfile=nwbfile, name=module_name, description=module_description)
```

Comment:
Nope. I tend to forget that GitHub has nice file diffing—so I keep comments assuming they're useful for review of what's been removed from the code.

If that's not actually useful, I'll start breaking that habit.

Reply:
Yeah, we typically only use comments for leaving long term notes (human readable, not usually code unless that's what is being communicated) to either communicate things via the git diff OR, more often, so that we can see it in the raw code view when coming back to a project after months-years (where there would not be open PRs to view diffs of)